src/production/phases/BundlePhase.ts

```diff
@@ -31,19 +31,19 @@ export async function BundlePhase(productionContext: IProductionContext) {
       router.generateSplitBundles(productionContext.splitEntries.entries);
     }
     await ctx.ict.resolve();
-    const bundles = await router.writeBundles();
+
+    const { bundles, manifest, onComplete } = await createRunResponse(ctx, router);

```

Comment:
not sure about this... 🤔

Reply:
me neither... Writing shouldn't be there